docs/4.0/docs/examples/sealctl-registry.md

```diff
@@ -0,0 +1,226 @@
+# Using sealctl manager registry image
+
+## Image save to local registry dir
+
+### default type
+
+default type can auto list images, using rootfs list images logic .ex charts,manifests,images dir.
+
+```
+registry images manager save to local dir by default type
+
+Usage:
+  sealctl registry save default [CONTEXT] [flags]
+
+Flags:
+      --arch string          pull images arch (default "arm64")
+      --data-dir string      registry data dir path (default "/var/lib/registry")
+  -h, --help                 help for default
+      --max-pull-procs int   maximum number of goroutines for pulling (default 5)
+
+Global Flags:
+      --debug                   enable debug logger
+      --root string             storage root dir (default "/var/lib/containers/storage")
+      --runroot string          storage state dir (default "/run/containers/storage")
+      --show-path               enable show code path
+      --storage-driver string   storage-driver (default "overlay")
+      --storage-opt strings     storage driver option
+```
+
+
+
+#### How to using
+
+```shell
+sealctl  registry  save default . 
+```
+
+### raw type
+
+raw type using images as image list.
+
+```
+registry images manager save to local dir by raw type
+
+Usage:
+  sealctl registry save raw [flags]
+
+Flags:
+      --arch string          pull images arch (default "arm64")
+      --data-dir string      registry data dir path (default "/var/lib/registry")
+  -h, --help                 help for raw
+      --images strings       images list
+      --max-pull-procs int   maximum number of goroutines for pulling (default 5)
+
+Global Flags:
+      --debug                   enable debug logger
+      --root string             storage root dir (default "/var/lib/containers/storage")
+      --runroot string          storage state dir (default "/run/containers/storage")
+      --show-path               enable show code path
+      --storage-driver string   storage-driver (default "overlay")
+      --storage-opt strings     storage driver option
+```
+#### How to using
+
+```shell
+sealctl  registry  save raw --images nginx
+```
+
+## Status registry
+
+need sealctl login or sealos login it.
+
+```
+registry status
+
+Usage:
+  sealctl registry status [flags]
+
+Examples:
+sealctl registry status
+
+Flags:
+  -h, --help   help for status
+      --json   output in JSON format
+
+Global Flags:
+      --debug                   enable debug logger
+      --root string             storage root dir (default "/var/lib/containers/storage")
+      --runroot string          storage state dir (default "/run/containers/storage")
+      --show-path               enable show code path
+      --storage-driver string   storage-driver (default "overlay")
+      --storage-opt strings     storage driver option
+```
+
+#### How to using
+```
+root@default-golang-0:~# sealctl  registry  status
```

Comment:
I think things like `root@default-golang-0:~#` does not provide useful information and should be replaced with a cleaner `$` just to show that we're in the command line. We should not simply copy from our terminal and paste it into the docs without any modification.

```suggestion
$ sealctl registry status
```